rtb/norme-di-progetto/src/gestione-configurazione.tex

```diff
@@ -25,26 +25,39 @@ \subsubsection{Versionamento}
 viene incrementata.
 \subsubsection{Repository}
     \subsubsubsection{Tecnologie e strumenti}
-    Per il progetto viene usato \emph{GitHub}, che a sua volta implementa il sistema di versionamento 
-    \emph{Git}. Il gruppo fa pieno uso delle funzionalità di \emph{GitHub} tra cui:
+    Per il progetto viene usato \emph{GitHub}$^{G}$, che a sua volta implementa il sistema di versionamento 
+    \emph{Git}$^{G}$. Il gruppo fa pieno uso delle funzionalità di \emph{GitHub} tra cui:
     \begin{itemize}
         \item \textbf{Issue:} Permette di discutere, tracciare e risolvere un problema nel progetto;
         \item \textbf{Milestone:} Permette di raggruppare e monitorare insieme una serie di issues aiutando a tracciare il progresso verso obiettivi del progetto;
         \item \textbf{Pull request:} Consente di proporre delle modifiche al codice e di richiederne la revisione da un altro membro.
     \end{itemize}
     \subsubsubsection{Struttura Repository}
-    Si è deciso di utilizzare un'unica repository contenente il codice sorgente per documentazione e \emph{PoC}.
-    All'interno della repository troviamo una cartella per ogni documento denominata \emph{nome-documento}. Inoltre la
-    repository è suddivisa in 3 branch principali:
+    Si è deciso di utilizzare un'unica repository contenente il codice sorgente per la documentazione e per il \emph{PoC}$^{G}$.
+    La repository è composta da 3 cartelle così divise:
+    \begin{itemize}
+        \item \textbf{Template:} contenente il template per il documento latex;
+        \item \textbf{Candidatura:} contenente i documenti relativi alla candidatura, come l'analisi dei capitolati,
+        la lettera di presentazione, il preventivo dei costi e assunzione impegni e infine i verbali.
+        \item \textbf{RTB:} contenente i documenti relativi alla revisione ed è suddivisa in tre sotto cartelle:
+        \begin{itemize}
+            \item \textbf{Documentazione interna}, all'interno della quale sono contenute le norme di progetto;
+            \item \textbf{Documentazione esterna}, che comprende i documenti tra cui Analisi dei requisiti, Piano di progetto, Piano di qualifica e il Glossario;
+            \item \textbf{Verbali}, che a loro volta sono suddivisi in interni ed esterni.
+        \end{itemize}
+    \end{itemize}
+    Ciascun documento segue la convenzione di nomenclatura \emph{Nome\_Documento} ad eccezione dei verbali che vengono 
```

Comment:
Mi pare che i nomi dei documenti fossero tutti in minuscolo, controlla nella sezione documentazione delle ndp